searxng/searxng
# Simple theme: minor adjustements

## What does this PR do?

* index page: margin top is 24% of the view port.
  avoid to scroll a small screen,
  center the content in the middle of the screen

* link to preferences at the same height same the input fields

* increase the category tab heighs

* increase the margin bottom of the query field

* reduce the margin to 0.4rem ( = 6px) around the h3 elements in the results.

## Why is this change important?

Better UI (?)

## How to test this PR locally?

* index page: resize vertically the windows, check 

* link to the preferences:
  * index page, check the position of the preferences icon
  * search something: the icon should be at the same place (still under the mouse cursor)
  * resize the window vertically: check the icon is at the same height same the input fields (#q, etc...)

* category tab: matter of taste...
  * search for images
  * display the detail of an image, check the detail doesn't over the search fields:
![image](https://user-images.githubusercontent.com/1594191/139299140-dd6a5d5c-5b41-4ad9-b90f-3d570378c100.png)

* check everything again with a phone and a tablet window size.

## Author's checklist



## Related issues
